index.html

```diff
@@ -96,9 +102,27 @@ <h2>Helpful Links </h2>
       </div>
     </section>
 
-<section class="testinomials">
-  <h2>Testimonials section</h2>
-
+<section class="testimonials">
+  <h2 class="subheading">Testimonials section</h2>
+ 
+  
+  <div class="slider-controls">
+    <i class="fa-solid fa-circle-arrow-left slider-control slider--prev"></i>
+    <i class="fa-solid fa-circle-arrow-right slider-control slider--next"></i>
+    <!-- <ion-icon name="arrow-dropleft" class="slider-control slider--prev"></ion-icon>
+    <ion-icon name="arrow-dropright" class="slider-control slider--next"></ion-icon> -->
+  </div>
+  
+  <div class="slider">
+     <!-- THIS IS JUST A PLACEHOLDER ===> Edit the testimonial.js file to add a new testimonial. Create a new object in the array of testimonials with the format. -->
```

Comment:
Appreciate the code comment here!

Reply:
I think it was necessary. Thank you

Reply:
Yes, if other people want to add a testimonial this will give them some direction. 🙂